### DISCUSSION TITLE
Some mini.surround key bindings not working?

### DISCUSSION THREAD

**Discussion by tommyalatalo** [LazyVim/LazyVim#432] [Q&A] [answered]
I just looked into the `mini.surround` keybindings while trying out the plugin, but it seems that some are not working, specifically when I do `gza"` when hovering over a word in normal mode I would expect to get quotes added around the word. This only works in visual mode for me, even though I checked keymaps with `:nmap` and with telescope's builtin keymaps picker, I can only see `gza` mapped to mini.surround's "Add surrounding" function for both normal and visual modes.

Is this something that's misconfigured on my side, or a lazyvim issue, or a mini.surround issue I wonder?

**Comment by dpetka2001** [comment, marked as answer]
After you type `a` you should also specify a text object upon which the surrounding will take effect. For more help see `:h text-objects`.

**Comment by tommyalatalo** [comment]
Of course, that makes sense, I have no idea why I didn't think of that.
The whole sequence is quite long then `gzaiw"` in order to surround a word, hmm, I'll have to try this and see if it's nice to work with or if I'll try to come up with something shorter.

Thanks for the quick reply!
